### PR TITLE
changelog-08-23-2024

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-docs-changelog-08-23-2024-neondatabase.vercel.app/docs/changelog/2024-08-23

### CODE REVIEW

File: content/changelog/2024-08-23.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  ![Branch created by column](/docs/relnotes/branch_created_by_column.png 'no-border')
+- The **Billing** page for Free Plan users now shows a percentage value for branch compute usage.
+  ![Compute branches metric](/docs/relnotes/compute_branches.png 'no-border')
+- We've added a new example to the Neon Console Quickstart showing how to connect to to your Neon database from NestJS. You can access the Quickstart from your Project Dashboard by selecting **Show quickstart**.
+  ![NestJS example](/docs/relnotes/nestjs.png)
+- The **Primary compute** column on the Branches page now shows your configured autoscaling range, replacing the `Autoscales` badge that was shown previously.
+  ![Autoscaling range](/docs/relnotes/autoscaling_range.png 'no-border')
+- We now support self-serve account deletion should you need to remove your Neon account for any reason. See [Delete your account](/docs/manage/accounts#delete-your-account) for details.
```
Comment (clarkbw) [inline content/changelog/2024-08-23.md:41]: This link doesn't work for me

Comment (bgrenon) [reply content/changelog/2024-08-23.md:41]: Let's remove the delete item for now. Not sure it's going to get released today.

Comment (bgrenon) [inline content/changelog/2024-08-23.md:41]:
```suggestion
```

File: content/changelog/2024-08-23.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  ![delete your account](/docs/relnotes/delete_account.png)
```
Comment (bgrenon) [inline content/changelog/2024-08-23.md:42]:
```suggestion
```

File: content/changelog/2024-08-23.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+<details>
+<summary>**Fixes & improvements**</summary>
+
+- Fixed an issue that prevented database and role names from being fully displayed in the **Settings** &#8594; **Default database and role** section on the Vercel integration drawer, accessed from the **Integrations** page in the Neon Console.
+- Queries saved to the Neon SQL Editor **Saved** list are now limited to 9 KB in length. A similar restriction was introduced Neon SQL Editor **History** list last week. While you can execute longer queries from the SQL Editor, any query exceeding 9 KB will be truncated when saved. A `-- QUERY TRUNCATED` comment is added at the beginning of these queries to indicate truncation.
```
Comment (bgrenon) [inline content/changelog/2024-08-23.md:48]:
```suggestion
- Queries saved to the Neon SQL Editor **Saved** list are now limited to 9 KB in length. A similar restriction was introduced in the Neon SQL Editor **History** list last week. While you can execute longer queries from the SQL Editor, any query exceeding 9 KB will be truncated when saved. A `-- QUERY TRUNCATED` comment is added at the beginning of these queries to indicate truncation.
```

File: content/changelog/2024-08-23.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- We updated the Drizzle Studio version that powers the **Tables** page in the Neon Console. This update addresses an issue where updating a column value in one row via the table editor updated the same column value in in other rows.
+- Fixed an issue in the Neon Console where some page labels in the sidebar were not highlighted when selected.
+- Fixed an issue that caused a `Something went wrong error` to appear briefly after deleting a project from the **Settings** page in the Neon Console.
```
Comment (bgrenon) [inline content/changelog/2024-08-23.md:51]: Do we need to mention this one?

**Review by bgrenon — APPROVED** [review]
Looks good. A couple small items, and let's remove account delete for now.

File: content/changelog/2024-08-23.md (hunk 1 above)
Comment (danieltprice) [reply content/changelog/2024-08-23.md]: Removed for today's changelog. Doesn't look like it will make it in.

**Pull request merged**